osi_trafficupdate.proto

```diff
@@ -13,13 +13,12 @@ package osi3;
 // participant models to provide updates to their position and
 // state back to the simulation environment.
 //
-// \note Note that for reasons of convenience and consistency, the
+// \note For reasons of convenience and consistency, the
 // updated information is provided as a MovingObject.  Certain fields
-// of this sub-message need not be set and will be ignored by the
-// simulation environment, because they are in fact static information.
+// of this sub-message are not required to be set and will be ignored by the
+// simulation environment, because they are static information.
 // Instead of creating a seperate message type for only the non-static
-// information, re-use of the existing message was considered more
-// appropriate.
+// information, re-use existing message.
```

Comment:
I actually think that this whole sentence should be dropped, since it gives rationale and is not part of the definition.